### ISSUE TITLE
Provisioning Datasource: InfluxDB values not working

### ISSUE THREAD

**Issue by auwaerter** [grafana/grafana#64795]
Hello Team,

i was quite unsure where to open this issue, as it is mainly something i would put into the Helm Charts directory.

Anyway, i've created the issue already there, but i'm not sure if it receives any attention.

=> [Link to Issue](https://github.com/grafana/helm-charts/issues/2243)

> ![image](https://user-images.githubusercontent.com/6923945/225255521-fec8effc-a86c-4e70-a73e-c276558cb1d8.png)

**Comment by j-sokol** [comment]
Looks like it is an error inside grafana, it's not picking up the `datasource` field from the grafana.ini config.
the datasource config:
```
  datasources.yaml: |
    apiVersion: 1
    datasources:
    - access: proxy
      database: Monitoring
      isDefault: true
...
```
but then when I check the `/api/datasources` endpoint, datasource field is empty:
``` 
- id: 1
  uid: P26B7E1BDE81CBED4
  orgId: 1
  type: influxdb
  typeName: InfluxDB
  typeLogoUrl: public/app/plugins/datasource/influxdb/img/influxdb_logo.svg
  access: proxy
  user: ""
  database: ""
...
```

**Comment by usmangt** [comment]
Hello @auwaerter and @j-sokol ,

Thanks for reporting this issue and providing v.valuable information in advance.

We have forwarded it to our team for further investigation. You will get an update once they have more information.

**Comment by itsmylife** [comment]
Hi @auwaerter @j-sokol  This (https://github.com/grafana/grafana/pull/62790) is going to fix this problem. The next release will have the fix.

**Comment by itsmylife** [comment]
The next version of v9.4.x and v9.5.x will have it.

I am closing this issue since we have a solution. If you think closing is a mistake feel free to open it. Let me know if I can help more.

**Comment by auwaerter** [comment]
Issue has been fixed for me with an update to v9.5.1
